test/e2e/dra/test-driver/app/kubeletplugin.go

```diff
@@ -135,13 +137,26 @@ func (ex *ExamplePlugin) IsRegistered() bool {
 	return status.PluginRegistered
 }
 
+// Block sets a flag to block Node[Un]PrepareResources
+// to emulate time consuming or stuck calls
+func (ex *ExamplePlugin) Block() {
+	ex.block = true
+}
+
 // NodePrepareResource ensures that the CDI file for the claim exists. It uses
 // a deterministic name to simplify NodeUnprepareResource (no need to remember
 // or discover the name) and idempotency (when called again, the file simply
 // gets written again).
 func (ex *ExamplePlugin) NodePrepareResource(ctx context.Context, req *drapbv1.NodePrepareResourceRequest) (*drapbv1.NodePrepareResourceResponse, error) {
 	logger := klog.FromContext(ctx)
 
+	// Block to emulate plugin stuckness or slowness
+	// By default the call will not be blocked as ex.block = false
```

Comment:
```suggestion
	// Block to emulate plugin stuckness or slowness.
	// By default the call will not be blocked as ex.block = false.
```

Reply:
Ah, I missed the dots.  Thanks for pointing out. done.